### ISSUE TITLE
[Suggestion] Check for Dependancies

### ISSUE THREAD

**Issue by BamsTheSergal** [soda3x/ArmaReforgerServerTool#48]
it would be nice if there was a way to check for missing dependancies

**Comment by soda3x** [comment]
Will look into it

**Comment by oIBRAHIMo1** [comment]
yes agree, maybe a drop down on each mod that says the dependent ones

**Comment by soda3x** [comment]
Not required anymore as dependencies are automatically downloaded from the workshop now
